bpf/netdev_config.h

```diff
@@ -25,3 +25,4 @@
 #define HANDLE_NS
 #define FROM_HOST
 #define ENCAP_IFINDEX 1
+#define POLICY_MAP cilium_policy_foo
```

Comment:
Why "foo"?

Reply:
This config file is only used during local builds, and this ensures that the POLICY_MAP name is defined to be "something". In actual deployments, `cilium-agent` will define this to be something sane when generating the BPF header files.